com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderCompilers.java

```diff
@@ -71,6 +71,7 @@ private ArrayList<ShaderDesc.Language> getPlatformShaderLanguages(ShaderDesc.Sha
                         shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLES_SM300);
                         shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLES_SM100);
                     }
+                    shaderLanguages.add(ShaderDesc.Language.LANGUAGE_WGSL);
```

Comment:
I don't see a very clean way to do that, it looks to me like the spirv is pretty much always used (since that is where the reflection data comes from). Do you think it just takes up too much room in the bundle or the processing is too slow?

Reply:
I was thinking more about the WGSL shaders, I'm not sure that we want to include the webgpu backend by default, so if you are not linking with the WGPU adapter there is no need to bundle the WSL shaders.

Reply:
We have that setup with vulkan, we will always use SPV in the pipeline, but we don't need to include those shaders into the bundle if it's not going to be used

Reply:
Seems reasonable, will of course make it impossible to repack it in there :) Does java know which adapters are currently enabled?

Reply:
We detect it by looking at the project settings (or rather, by looking at the "app manifest"):
https://github.com/defold/defold/blob/040e2a361bd9d6a88f068cb225bfc8445df19484/com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java#L1366

We need some adjustments in the editor to add the option to the editors manifest tool, so for the "technical preview" of this feature I suggest we start by adding the debug flag similar to SPIR-v and after this is merged we can do a follow-up PR with editor and extender (build server) changes :)

Reply:
Will do.